sdk/servicenetworking/Azure.ResourceManager.ServiceNetworking/src/Azure.ResourceManager.ServiceNetworking.csproj

```diff
@@ -5,4 +5,7 @@
     <Description>Azure Resource Manager client SDK for Azure resource provider ServiceNetworking.</Description>
     <PackageTags>azure;management;arm;resource manager;servicenetworking</PackageTags>
   </PropertyGroup>
+  <ItemGroup>
+    <PackageReference Update="Azure.Core" Version="1.31.0" />
+  </ItemGroup>
```

Comment:
I believe this is not necessary, could we please revert this?

Reply:
sure, reverted